the-collab-lab/tcl-28-smart-shopping-list
# Ks ia router

## Description

- Added react-router-dom as dependecy
- Created shell AddItem and ItemsList components
- Wrapped App in BrowserRouter
- Added Routes to AddItem and ItemsList
- Created Footer component 
- Added links in Footer component to Add Item and List views
- Styled basics of Footer component (sticks on bottom, bold links when on current page)

## Related Issue

#2 

## Acceptance Criteria


## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

Nothing!

### After

![add-item-page](https://user-images.githubusercontent.com/56979810/125338194-e6c29280-e31d-11eb-92c8-1bd6cc54b0f1.JPG)

![items-list-page](https://user-images.githubusercontent.com/56979810/125338111-ca265a80-e31d-11eb-8817-963974043796.JPG)



## Testing Steps / QA Criteria



- From your terminal, pull down this branch
- `npm install`
- `npm start`
- Click the links to toggle between views
- Check that the corresponding link to the view you are on is bolded (except on initial '/' view, that is a placeholder)
